gcloud/credentials.py

```diff
@@ -72,8 +73,34 @@ def get_credentials():
     return client.GoogleCredentials.get_application_default()
 
 
+def get_for_service_account_json(private_key_path, scope=None):
```

Comment:
Cool, I'll change to JSON credentials since the file holds a JSON object, including more than just a private key (in PEM format).